Formula/intermodal.rb

```diff
@@ -0,0 +1,29 @@
+class Intermodal < Formula
+  desc "User-friendly and featureful command-line BitTorrent metainfo utility"
+  homepage "https://imdl.io"
+  url "https://github.com/casey/intermodal/archive/refs/tags/v0.1.12.tar.gz"
+  sha256 "cd62894e519dc5aa0284a5f48aab86e1a45c3bc96b8a5481741adb6960d4751a"
+  license "CC0-1.0"
+  head "https://github.com/casey/intermodal.git"
+
+  depends_on "help2man" => :build
+  depends_on "openssl@1.1" => :build
+  depends_on "rust" => :build
+
+  def install
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
```

Comment:
Does this link statically with OpenSSL? Can we get it to link dynamically instead? That way this formula isn't stuck using an outdated version of OpenSSL that may or may not have outstanding CVEs.